Mathlib/Analysis/PSeries.lean

```diff
@@ -223,7 +223,7 @@ theorem summable_nat_pow_inv {p : ℕ} :
 if and only if `1 < p`. -/
 theorem summable_one_div_nat_pow {p : ℕ} :
     Summable (fun n => 1 / (n : ℝ) ^ p : ℕ → ℝ) ↔ 1 < p := by
-  -- porting note (#10745): was `simp`
+  -- porting note (#10745): added two simp lemmas
```

Comment:
Clarified this porting note; the important part is that these lemmas need to be supplied to simp now. (The former is tagged simp, the latter is not.)

Reply:
I originally read this as you wrote two new lemmas and tagged them with `simp`. Perhaps "supplied" is a clearer verb.